app/src/editor/canvas/components/ModuleSearch.pcss

```diff
@@ -63,18 +62,31 @@
   color: #A3A3A3;
 }
 
-.ModuleSearch .Input input {
+.ModuleSearch .Header svg:hover path {
+  fill: #525252;
```

Comment:
Expand/collapse toggle is built using a line. `fill` breaks it a little bit.

![May-03-2019 10-23-39](https://user-images.githubusercontent.com/320066/57126563-a8200d80-6d8d-11e9-84b4-441d714058e1.gif)

Reply:
Imma have to request changes for this! 🐽

Reply:
ahh you're right